cns/cnsclient/apiclient.go

```diff
@@ -4,5 +4,7 @@ import "github.com/Azure/azure-container-networking/cns"
 
```

Comment:
Since this is called in CRD Reconcile, this should always call CreateNetworkContainerRequest)